x-pack/plugins/observability_solution/logs_explorer/common/locators/logs_explorer/logs_explorer_locator.ts

```diff
@@ -21,16 +21,21 @@ export class LogsExplorerLocatorDefinition implements LocatorDefinition<LogsExpl
   constructor(protected readonly deps: LogsExplorerLocatorDependencies) {}
 
   public readonly getLocation = (params: LogsExplorerLocatorParams) => {
-    const { dataset } = params;
+    const { dataset, columns } = params;
     const dataViewSpec: DataViewSpec | undefined = dataset
       ? {
           id: dataset,
           title: dataset,
         }
       : undefined;
 
+    const discoverStyleColumns = columns?.map((column) => {
```

Comment:
`discoverStyleColumns` doesn't mean much about what this variable holds, right? Can you improve that?

Reply:
I would remove the `style` part as its confusing. The idea to add discover is, this locator is actually a `LogExplorerLocator` but is created with an instance of `DiscoverLocator`.

Hence we need to strip down the `columns` props as per DiscoverLocator here. Hence the name with `discoverPrefix` gives the idea that a `LogExplorer` style column is being changed to `Discover` style column